### ISSUE TITLE
Log Cleanup

### ISSUE THREAD

**Issue by ObaidGH** [AlexxEG/YouTube-Downloader#37]
Unable to auto cleanup log

MaxLogAge is set to 60 seconds

``` c#
public static TimeSpan MaxLogAge = new TimeSpan(0, 0, 0,60);
```

**Comment by AlexxEG** [comment]
It's working correctly for me. How does it not work for you?

**Comment by AlexxEG** [comment]
Does this still not work? I've done some testing and it seems to work correctly for me.

**Comment by ObaidGH** [comment]
First of all, how does it work?

**Comment by ObaidGH** [comment]
Hey, it works now, just made a little change.

From this:
```c#
if (age >= MaxLogAge )
```
To this:
```c#
if (MaxLogAge >= age)
```

It also works this way
```c#
if (age <= MaxLogAge)

**Comment by AlexxEG** [comment]
It only supposed to delete logs that are OLDER than `MaxLogAge`. The default is 14 days, so any logs that's older than that gets deleted.

14 days might be too long though, maybe 3 days would be better.

Anyway, your edit basically just deletes all files regardless of age. Unless you don't start the program for 14 days so that the files then are older than `MaxLogAge`

**Comment by ObaidGH** [comment]
Ok, it works fine now, I set the age limit to 3 days and changed the system date accordingly

**Comment by ObaidGH** [comment]
Nope, it's not, minutes or seconds specified don't work

**Comment by ObaidGH** [comment]
Nope, it's still doesn't work

**Comment by AlexxEG** [comment]
What exactly are you trying to do?

**Comment by ObaidGH** [comment]
I set the time span to 1 minute
```c#
public static TimeSpan MaxLogAge = new TimeSpan(0, 0, 1, 0);

**Comment by AlexxEG** [comment]
I just tested it with 1 minute myself, and it works correctly.

What I did:

1. Found a random YouTube link and retrieved info in my program (no need to start the download to test)
2. A .json file gets created at `%localappdata%\YouTube Downloader\json`
3. Wait 1 minute and restart the program
4. File gets deleted

Maybe you're expecting it to delete while program is still running? It only checks on startup.
